### PR TITLE
Automatic updating of buildtools version.

### PR DESCRIPTION
**Pull request by eerhardt**
With this change, BuildTools will automatically be updated to the latest available version.

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -83,7 +84,11 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)projectn-tfs-testilc/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(ProjectNTfsTestILCCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
-
+    <RemoteDependencyBuildInfo Include="BuildTools">
+      <BuildInfoPath>$(BaseDotNetBuildInfo)buildtools/$(DependencyBranch)</BuildInfoPath>
+      <CurrentRef>$(BuildToolsCurrentRef)</CurrentRef>
+    </RemoteDependencyBuildInfo>
+
```
Comment (dagood) [inline dependencies.props]: Super nitpick: trailing whitespace.

Comment (eerhardt) [reply dependencies.props]: Fixed.  I also updated BuildTools to the latest version to ensure the auto-updating will continue to work.  As I see some errors updating it in coreclr.

**Comment by mmitche** [comment]
@dotnet-bot test this please (accidentlaly deleted VMs while in use)

**Comment by eerhardt** [comment]
Test Linux x64 Release Build

**Comment by eerhardt** [comment]
test Linux x64 Release Build

**Pull request merged**